### ISSUE TITLE
Crashes during the bakery quest

### ISSUE THREAD

**Issue by Sometwat** [Kyuuhachi/Azure-Vitality#9]
After the song, the game crashes. Even giving me a suggestion to report the issue to blah blah. Tried it a few times with different characters and installed the mods again, no dice.

**Comment by Kyuuhachi** [comment]
I'm currently on vacation, but I can look into it sometime next week. In the meantime, what game version do you have, and do you use any other mods? A save file would be useful too.

**Comment by Sometwat** [comment]
I'm not sure what you mean by game version? Steam, if that's what you mean. Vitality portraits and more portraits, both the recent versions. I'll also need to figure out how to send over a save, but I will.

**Comment by Sometwat** [comment]
Ok, I'm a bit at a loss for how to share my save

**Comment by Kyuuhachi** [comment]
Game version should be in the bottom corner on the title screen. Saves should be in `%USERPROFILE%\Saved Games\FALCOM\Ao\`. Just zip up the folder with the right number, and you can then drag it over the text area down here to upload.

**Comment by Sometwat** [comment]
Ah, that version. V1.1.18. I'll try sending over the save in a moment

**Comment by Kyuuhachi** [comment]
Also remember to install Vitality Portraits after More Portraits, not before.

**Comment by Sometwat** [comment]
Good there, don't worry

**Comment by Sometwat** [comment]
Anyway, where is the %user so on and so? Is it in the game files you can access by the "browse local files option?"

**Comment by Kyuuhachi** [comment]
Just write it in the path field in explorer (the file manager), Windows will automatically substitute it. Oh and isn't there an open folder key in the save/load menu?

**Comment by Sometwat** [comment]
![image](https://github.com/Kyuuhachi/Azure-Vitality/assets/174006646/62cc7e52-a66d-4e23-8ea8-303ca49605d1)
This is all I see? I'll try the path field.. thing

**Comment by Sometwat** [comment]
Ok...
[data0069.zip](https://github.com/user-attachments/files/16011991/data0069.zip)
Will this do?

**Comment by Kyuuhachi** [comment]
I'll try it when I'm back.

**Comment by Sometwat** [comment]
Ok, thanks!

**Comment by Sometwat** [comment]
Ok, a strange update.. I tried doing it after changing which display I was using (to my NVIDIA GPU) and that might be why it worked? I'm not totally sure.

**Comment by Sometwat** [comment]
I tried doing the quest with integrated graphics again and it crashed. Seems to be the cause..

**Comment by Kyuuhachi** [comment]
That sounds difficult to debug. The scripts aren't supposed to be doing anything gpu-related. At what exact point does it crash?

**Comment by Sometwat** [comment]
Right.. it's always after the song, at the same spot. I'm no real help here, but I did get past it at least.

**Comment by Sometwat** [comment]
It almost seems like it crashes when trying to show the textbox. Can't say though.

**Comment by Kyuuhachi** [comment]
Would it be possible to record a video?

**Comment by Sometwat** [comment]
It should be... Give me a bit.

**Comment by Sometwat** [comment]
Sorry, what's a good way to send it? Keeps saying the file is too big

**Comment by Sometwat** [comment]
[](url)

https://github.com/Kyuuhachi/Azure-Vitality/assets/174006646/42b6f5c6-9483-423c-8f16-377f07c085d1

**Comment by Sometwat** [comment]
Nevermind I just bumbled into this

**Comment by Angko18** [comment]
i have the exact same issue ,any clue how to solve it?

**Comment by RyuOgi** [comment]
2026/07/15: Hello Folks, I had the same issue where the game would crash after the song finishes and the first dialogue starts.
What worked for me was: First, verify files in Steam, this removed all mods i had. Second, installed only "More Portraits v1.4.0 [Latest]". Thirdly, and probably most important, started Windows 11 in Safe mode with network services only (via msconfig). Steam will start, make some noise on how some services "must be installed", which are disabled due to the safe mode.
There you start the game, load your save, do the request and it will advance as expected without crashing. There you save again, and boot back to normal (via msconfig, simply select the normal boot option). 

Screenshot:
[https://imgur.com/a/NjboLid](url)

Thanks for the Mod @Kyuuhachi !

**Comment by Vetches** [comment]
Adding another datapoint after @RyuOgi's revelatory discovery about Safe Mode - I'm on Windows 10, didn't verify the files or remove any mods (just had the portraits + EVO quests installed), and booting into Safe Mode with Network services and opening the game on Steam worked fine! The cutscene worked, quest was completed!
